Xamarin.Forms.Platform.UAP/ImageRenderer.cs

```diff
@@ -189,5 +195,34 @@ protected async Task UpdateSource()
 				Element.SetIsLoading(false);
 			}
 		}
+
+		void UpdateImageSearchDirectory()
+		{
+			if (_disposed || Element == null || Control == null)
+			{
+				return;
+			}
+
+			var imageSearchDirectory = Application.Current.OnThisPlatform().GetImageSearchDirectory();
+
+			if(!string.IsNullOrEmpty(imageSearchDirectory))
+			{
+				var source = Element.Source;
+
+				if (source is FileImageSource fileSource && fileSource != null)
+				{
+					var filePath = fileSource.File;
+
+					var directory = Path.GetDirectoryName(filePath);
+
+					if (string.IsNullOrEmpty(directory) || !Path.GetFullPath(directory).Equals(Path.GetFullPath(imageSearchDirectory)))
```

Comment:
So if the file appears to live at the search path then we don't combine the path with the search path. Means we can't do `assets\assets` right? Is there a reason to be backward compatible after someone has set the platform specific? Why not just drop this `if` and always combine? Certainly would be easier to document: 
> We prepend the search path to all asset paths.